docs/TuxRouter.md

```diff
@@ -0,0 +1,5 @@
+#Tux/Router
+
+## <a id="Premise"></a>Premise [#](#Premise)
+>The Tux Router simply exposes the functionality of the amazing **react-router** npm and allows it to be easily accessible in your Tux projects. This documentation is borrowed from the **react-router** docs and is only slightly adjusted. To read more about the fabulous **react-router** visit the [**react-router** github page.](https://github.com/rackt/react-router)
+
```

Comment:
Need to mention how `tux` breaks up the `react-router` into individual modules.  Thus, in `tux` you require the `tux/Router/ROUTER COMPONENT`.  Maybe we can have links to each corresponding component within the `react-router`.